proto/prehog/v1alpha/teleport.proto

```diff
@@ -628,6 +659,10 @@ message SubmitEventRequest {
 
     UIIntegrationEnrollStartEvent ui_integration_enroll_start_event = 44;
     UIIntegrationEnrollCompleteEvent ui_integration_enroll_complete_event = 45;
+
+    EditorChangeEvent editor_change_event = 46;
+
+    BotCreateEvent bot_create = 47;
```

Comment:
Are you going to extend the `ConvertUsageEvent` in another PR ?

Reply:
This event isn't submitted externally to the auth server, so I don't think `ConvertUsageEvent` needs extending ?